scripts/globals/zone.lua

```diff
@@ -384,6 +384,143 @@ dsp.zone =
     WALK_OF_ECHOES_P                = 298,
 }
 
+dsp.expansionZone = {}
+dsp.expansionZone.ORIGINAL_ROTZ = {
+    [dsp.zone.WEST_RONFAURE]                  = true,
+    [dsp.zone.EAST_RONFAURE]                  = true,
```

Comment:
I think this could be shortened by a lot - do you really need to explicitly check the zone by expansion or can you just check if the region or that the region is NOT one form the other expansions? I think the only other helm area's would be wotg and cop. https://github.com/DarkstarProject/darkstar/blob/master/src/map/zone.h#L347

(might be able to bind contententtype and use it depending on if cop area's round as middle lands or "other", but I did not look..prolly would need to region it at a min..In the future I'd like to do something outside of lua for zone expansion info but that is outside the scope of this PR)